app/javascript/components/provider-form/index.jsx

```diff
@@ -162,7 +162,11 @@ const ProviderForm = ({
       const { endpoints: _endpoints = { default: {} }, authentications: _authentications = {}, ...rest } = pick(_data, toSubmit);
       // Convert endpoints and authentications back to an array
       const endpoints = Object.keys(_endpoints).map((key) => ({ role: key, ..._endpoints[key] }));
-      const authentications = Object.keys(_authentications).map((key) => ({ authtype: key, ..._authentications[key] }));
+      let authentications = Object.keys(_authentications).map((key) => ({ authtype: key, ..._authentications[key] }));
+
+      if (authentications.length === 0) {
```

Comment:
Will this break the scenario where you have authentications and then remove them and save?  I know we might not allow it in the first place but it seems like the submit should fail with missing authentication and not put it back.

Reply:
Offline discussion... if auth is modified, this issue doesn't happen.

I'm fine with this approach but being naive to the code, I would have expected `modified` to include the auth if it was modified.  Then the conditional could be something like:

```
if auth_modified
  # use modified auth
  let auth = Object.keys(_authentications)...
else
  # use original/unchanged auth
  let auth = Object.keys(initialValues.authentication...
```